importer/markdown/src/test/java/org/itsallcode/openfasttrace/importer/markdown/ITMarkdownImporter.java

```diff
@@ -0,0 +1,247 @@
+                + "\n" + ID2 + "\n" //
+                + "# Irrelevant Title";
+    }
+
+    @Test
+    void testSingleNeeds()
+    {
+        final String singleNeedsItem = "`foo~bar~1`\n\nNeeds: " + NEEDS_ARTIFACT_TYPE1;
+        final List<SpecificationItem> items = runImporterOnText(singleNeedsItem);
+        assertThat(items.get(0).getNeedsArtifactTypes(), contains(NEEDS_ARTIFACT_TYPE1));
+    }
+
+    @Test
+    void testFindLegacyRequirement()
+    {
+        final String completeItem = createCompleteSpecificationItemInLegacyMarkdownFormat();
+        assertThat(runImporterOnText(completeItem),
+                AutoMatcher.contains(SpecificationItem.builder().id(SpecificationItemId.parseId(LEGACY_ID))
+                        .title("Requirement Title")
+                        .status(ItemStatus.PROPOSED)
+                        .comment("Comment" + NL + "More comment")
+                        .description("Description" + NL + NL + "More description")
+                        .rationale("Rationale" + NL + "More rationale")
+                        .addNeedsArtifactType("artA").addNeedsArtifactType("artB")
+                        .addCoveredId(SpecificationItemId.parseId(LEGACY_COVERED_ID1))
+                        .addCoveredId(SpecificationItemId.parseId(LEGACY_COVERED_ID2))
+                        .addDependOnId(SpecificationItemId.parseId(LEGACY_DEPENDS_ON_ID1))
+                        .addDependOnId(SpecificationItemId.parseId(LEGACY_DEPENDS_ON_ID2))
+                        .addTag("Tag1").addTag("Tag2")
+                        .location("file name", 2)
+                        .build()));
+    }
+
+    // [utest->dsn~md.needs-coverage-list~2]
+    private String createCompleteSpecificationItemInLegacyMarkdownFormat()
+    {
+        return "# " + TITLE //
+                + "\n" //
+                + "`" + LEGACY_ID + "`" //
+                + "\n" //
+                + "\nStatus: proposed\n" //
+                + "\nDescription:\n" + DESCRIPTION_LINE1 + "\n" //
+                + DESCRIPTION_LINE2 + "\n" //
+                + DESCRIPTION_LINE3 + "\n" //
+                + "\nRationale:\n" //
+                + RATIONALE_LINE1 + "\n" //
+                + RATIONALE_LINE2 + "\n" //
+                + "\nDepends:\n\n" //
+                + "  + `" + LEGACY_DEPENDS_ON_ID1 + "`\n" //
+                + "  - `" + LEGACY_DEPENDS_ON_ID2 + "`\n" //
+                + "\nCovers:\n\n" //
+                + "  * `" + LEGACY_COVERED_ID1 + "`\n" //
+                + " + `" + LEGACY_COVERED_ID2 + "`\n" //
+                + "\nComment:\n\n" //
+                + COMMENT_LINE1 + "\n" //
+                + COMMENT_LINE2 + "\n" //
+                + "\nNeeds:\n" //
+                + "   * " + NEEDS_ARTIFACT_TYPE1 + "\n"//
+                + "+ " + NEEDS_ARTIFACT_TYPE2 + "\n" //
+                + "\nTags: " + TAG1 + ", " + TAG2;
+
+        // + "\nTags:\n" //
+        // + " * " + TAG1 + "\n" //
+        // + " + " + TAG2 + "";
```

Comment:
```suggestion

        // + "\nTags:\n" //
        // + " * " + TAG1 + "\n" //
        // + " + " + TAG2 + "";
```

Please remove commented code.